src/main/java/hudson/plugins/locale/PluginImpl.java

```diff
@@ -66,13 +67,18 @@ public void configure(StaplerRequest req, JSONObject jsonObject)
             throws IOException, ServletException, FormException {
         setSystemLocale(jsonObject.getString("systemLocale"));
         ignoreAcceptLanguage = jsonObject.getBoolean("ignoreAcceptLanguage");
+        userPrefer = jsonObject.getBoolean("userPrefer");
         save();
     }
 
     public boolean isIgnoreAcceptLanguage() {
         return ignoreAcceptLanguage;
     }
 
+    public boolean isUserPrefer() {

```

Comment:
It would be great to have a more explicit method name, bonus points for Javadoc

```suggestion
    public boolean isAllowUserPreferences() {
```